source/language/openpulse.rst

```diff
@@ -235,7 +235,7 @@ samples provides optimization opportunities that wouldn't be available otherwise
 .. code-block:: javascript
 
    // arbitrary complex samples
-   waveform arb_waveform = [1+0*j, 0+1*j, 1/sqrt(2)+1/sqrt(2)*j];
+   waveform arb_waveform = [1+0im, 0+1im, 1/sqrt(2)+1/sqrt(2)im];
```

Comment:
Should these use `*Im`?  Same deal with line 314. 

Also, should we add some statement about the complex number support still being in active development (see [here](https://github.com/Qiskit/openqasm/pull/301))?

Reply:
Added comment in 4f13bd9.